crates/fluence-app-service/src/service.rs

```diff
@@ -144,23 +143,12 @@ impl AppService {
         let base_dir = &config.service_base_dir;
         let service_dir = base_dir.join(&service_id);
         create(&service_dir)?;
-
-        let local_dir = service_dir.join(SERVICE_LOCAL_DIR_NAME);
-        create(&local_dir)?;
-
-        let tmp_dir = service_dir.join(SERVICE_TMP_DIR_NAME);
-        create(&tmp_dir)?;
-
-        let local_dir = local_dir.to_string_lossy().to_string();
-        let tmp_dir = tmp_dir.to_string_lossy().to_string();
-
-        let mut preopened_files = HashSet::new();
-        preopened_files.insert(PathBuf::from(local_dir.clone()));
-        preopened_files.insert(PathBuf::from(tmp_dir.clone()));
+        create(&service_dir.join(SERVICE_LOCAL_DIR_NAME))?;
+        create(&service_dir.join(SERVICE_TMP_DIR_NAME))?;
```

Comment:
plz add comment on why you create these directories

Reply:
Maybe you don't even need to create them? Maybe they should be created in `set_service_base_dir` function? or after it.

Like, all directories mentioned in the config must be created, I guess.

Reply:
Why do so? By creating them i will change the behaviour. I try to add some security wihout changing old behaviour